app/src/main/res/layout/fragment_about.xml

```diff
@@ -15,16 +15,14 @@
         android:paddingBottom="@dimen/activity_vertical_margin">
 
         <ImageView
-            android:id="@+id/logo"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="center_horizontal"
             android:layout_marginBottom="8dp"
-            android:contentDescription="TODO"
+            android:contentDescription="NewPipe Logo"
```

Comment:
I think this should be made translatable, or maybe instead just use the untraslatable app name string

Reply:
Very well.